### ISSUE TITLE
Error installing vue-cli-plugin-vuetify@0.4.0

### ISSUE THREAD

**Issue by WitteStier** [vuetifyjs/vue-cli-plugin-vuetify#49]
Ubuntu@18.02
vue/cli@3.0.5

NPM, Babel, TS, PWA, TSLint, Jest, Cypress

```
📦  Installing vue-cli-plugin-vuetify...

+ vue-cli-plugin-vuetify@0.4.0
updated 1 package and audited 48036 packages in 8.224s
found 0 vulnerabilities

✔  Successfully installed plugin: vue-cli-plugin-vuetify

 ERROR  Error: ENOENT: no such file or directory, scandir '/home/wittestier/Web/elkerbout/client/node_modules/vue-cli-plugin-vuetify/node_modules/vuetify/src/locale'
Error: ENOENT: no such file or directory, scandir '/home/wittestier/Web/elkerbout/client/node_modules/vue-cli-plugin-vuetify/node_modules/vuetify/src/locale'
    at Object.fs.readdirSync (fs.js:921:18)
    at Object.<anonymous> (/home/wittestier/Web/elkerbout/client/node_modules/vue-cli-plugin-vuetify/util/locales.js:24:21)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/home/wittestier/Web/elkerbout/client/node_modules/vue-cli-plugin-vuetify/prompts.js:6:17)

```

**Comment by WitteStier** [comment]
Confirmed that vue-cli-plugin-vuetify@0.4.4 is working under windows. I will retry it on Ubuntu when i'm at home. But i guess its fixed by a5bd7bf0b60a4d1e1aed834894e54e62a841aa64.

Thanks in advanced.

**Comment by jacekkarczmarczyk** [comment]
Thank you, create a new issue (with reproduction steps) if there's still a bug
